Explorer/Assets/Scripts/CrdtEcsBridge/WorldSynchronizer/CommandBufferSynchronizer/SDKComponentCommandBufferSynchronizer.cs

```diff
@@ -49,9 +50,17 @@ public SDKComponentCommandBufferSynchronizer(IComponentPool<T> componentPool)
                     }
                     else
                     {
-                        ref T pointerToPrevObj = ref world.Get<T>(entity);
-                        componentPool.Release(pointerToPrevObj);
-                        pointerToPrevObj = c;
+                        try
```

Comment:
What is the reason of handling the exception here? Using TryGetRef could be useful in this case?:
```
ref T pointerToPrevObj = ref world.TryGetRef<T>(entity, out bool hasComponent);
```

Reply:
no reason, debug stuff